buddyboss/buddyboss-platform
# #3200 : Fixed unknown title text when hit the desktop notification

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #3200 .

### How to test the changes in this Pull Request:

1. Go to Buddyboss settings > notifications
2. Enable Show in Browser Tab the option
3. Move on forum discussion of any page and get the desktop notification from another user or browser
4. Check the title in the current tab ( or check the below-attached screenshot)


### Proof Screenshots or Video
https://www.awesomescreenshot.com/video/6194299?key=db37f00ee68c49e36ec74f172f12961a


### Other information:




### Changelog entry

> Fixed unknown title text when hit the desktop notification.
